Library/Formula/irssi.rb

```diff
@@ -21,7 +27,7 @@ class Irssi < Formula
   depends_on "openssl" => :recommended
   depends_on "dante" => :optional
 
-  # Fix Perl build flags and paths in man page
+  # Make paths in man page Homebrew-specific
```

Comment:
Can you switch this to using an `inreplace` instead (and ideally file an upstream issue to complain about `configure` not processing this file correctly)? Thanks!